srcpkgs/pam-base/files/system-login

```diff
@@ -16,6 +15,5 @@ session    include    system-auth
 session    optional   pam_motd.so          motd=/etc/motd
 session    optional   pam_mail.so          dir=/var/mail standard quiet
 -session   optional   pam_elogind.so
--session   optional   pam_ck_connector.so  nox11
 session    required   pam_env.so
-session    required   pam_lastlog.so       silent
+-session   optional   pam_lastlog.so       silent
```

Comment:
Upon further testing, I recommend we just remove the `pam_lastlog` module from the default configuration. According to [`pam_lastlog(8)`](https://man.voidlinux.org/pam_lastlog), this module will record the current login in `/var/log/lastlog` and, if `silent` is not included, print information about the most recent prior login.

But every time I log in, I see a "last login" message even in `silent` mode, and it always shows the time and terminal information for the *current*, rather than a prior, login. If I remove the `silent` directive, I see *two* "last login" messages, the first of which shows the expected information about an actual prior login and the second showing information about the current login.

What is going on? It turns out that [`login(1)`](https://man.voidlinux.org/man1/login.1) also records to `/var/log/lastlog` and, if a `.hushlogin` file is not present, prints information about the most recent prior login.

Thus what seems to be happening is:
1. `agetty` takes user credentials and runs them through PAM.
2. After successful authentication, `pam_lastlog` reads `/var/log/lastlog` if not in `silent` mode and prints the last login for the authenticated user. Then, it records the current login information in the same file.
3. `agetty` starts `login` to begin the login process.
4. `login` effectively repeats step 2, but the last login it sees is the newly created record from `pam_lastlog`, so it prints that, then makes a duplicate record in `/var/log/lastlog`.

Cutting `pam_lastlog` out of this loop produces the expected behavior: I see information about the most recent prior login attempt when logging in. Note also the quote from the `pam_lastlog` manual page:
> Some applications may perform this function themselves. In such cases, this module is not necessary.
That is most definitely the case here.

Reply:
Oh my god that's such a maze. Thanks for looking deeply into it!

It definitely makes sense to remove this from the config, then.

Reply:
Done! I left a quick little explanation in the commit message, let me know if you feel more is necessary.

Reply:
I think this looks good.